docs/how-to/datapath-only-mode.rst

```diff
@@ -0,0 +1,112 @@
+
+Because the MicroOVN enforces encrypted communication between OVN services, we need
+to ensure that OVN services running on the MicroOVN cluster and on the external OVN
+central cluster are mutually trusted.
+
+Obtain intermediate CA certificate and private key from the CA that issued certificates
+for the external OVN central cluster and place it in the ``/var/snap/microovn/common/``
+on the ``node-1``, under names ``ca.cert`` and ``ca.key``.
+
+Bootstrap MicroOVN cluster
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+First we bootstrap the MicroOVN cluster on the ``node-1`` and generate tokens, so that
+``node-2`` and ``node-3`` can join.
+
+.. code-block::
+
+   $ sudo microovn init
+   Please choose the address MicroOVN will be listening on [default=10.75.224.213]:
+   Would you like to create a new MicroOVN cluster? (yes/no) [default=no]: yes
+   Please select comma-separated list services you would like to enable on this node (central/chassis/switch) or let MicroOVN automatically decide (auto) [default=auto]: switch,chassis
+   Please choose a name for this system [default=node-1]:
+   Would you like to define a custom encapsulation IP address for this member? (yes/no) [default=no]:
+   Would you like to provide your own CA certificate and private key for issuing OVN TLS certificates? (yes/no) [default=no]: yes
+   Please enter the path to the CA certificate file: /var/snap/microovn/common/ca.cert
+   Please enter the path to the CA private key file: /var/snap/microovn/common/ca.key
+   Would you like to add additional servers to the cluster? (yes/no) [default=no]: yes
+   What's the name of the new MicroOVN server? (empty to exit): node-2
+   <NODE_2_TOKEN>
+   What's the name of the new MicroOVN server? (empty to exit): node-3
+   <NODE_3_TOKEN>
+
+Then we join the cluster with ``node-2`` and ``node-3``
+
+.. code-block::
+
+   $ sudo microovn init
+   Please choose the address MicroOVN will be listening on [default=10.75.224.175]:
+   Would you like to create a new MicroOVN cluster? (yes/no) [default=no]: no
+   Please select comma-separated list services you would like to enable on this node (central/chassis/switch) or let MicroOVN automatically decide (auto) [default=auto]: chassis,switch
+   Please enter your join token: <NODE_TOKEN>
+   Would you like to define a custom encapsulation IP address for this member? (yes/no) [default=no]:
+
+External central configuration
+------------------------------
+
+Now that the MicroOVN cluster is deployed without the central services, we can configure
+it to connect to the external OVN central cluster. On any of the nodes, run following
+command:
+
+.. code-block::
+
+   sudo microovn config set ovn.central-ips "10.0.0.1,10.0.0.2,10.0.0.3"
+
+Verification
+------------
+
+To verify that the configuration was successfully applied, we can check that our
+chassis successfully registered with the OVN Southbound database.
+
+.. code-block::
+
+   $ sudo microovn.ovn-sbctl show
```

Comment:
No terminal prefix in code-blocks, make two code-blocks instead, one with the command, another with the output/interaction.